cmd/common.go

```diff
@@ -166,3 +168,19 @@ func Max(a, b int) int {
 	}
 	return b
 }
+
+// GetSobject retrieves a sobject through SDKMS client.
+// It takes a keyID as a parameter and returns a pointer to a Sobject.
+// If the keyID is empty, it will return an error.
+// If an error occurs during the retrieval process, the function will log a fatal error and exit.
+func GetSobject(kid *string) *sdkms.Sobject {
+	client := sdkmsClient()
+	client.Auth = sdkms.APIKey(apiKey)
```

Comment:
I'm not very happy with this function reading the global `apiKey` variable. But I guess it's fine here.

Reply:
Agree, this is also one of todo tasks.
These global variables makes it hard to write tests.